react/features/deep-linking/components/DeepLinkingMobilePage.web.js

```diff
@@ -10,6 +10,7 @@ import { DialInSummary } from '../../invite';
 
 import { _TNS } from '../constants';
 import { generateDeepLinkingURL } from '../functions';
+import { renderPromotionalFooter } from '../../deep-linking/renderPromotionalFooter';
```

Comment:
Actually this needs to be `../renderPromotionalFooter'.

Reply:
You are correct, oopsie :)